<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-240</x>
      <y>440</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperaturas:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="temp_default">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
   </widget>
   <widget class="QLabel" name="termp_defecto">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Celsius</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Kelvin:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Newton:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>136</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Fahrenheit:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conversor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Convertir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingrese la temperatura en Celsius</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp_Fahrenheit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>640</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp_Newton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>671</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp_Kelvin">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>691</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eleccion_temp">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Ingrese que temperatura desea usar como referencia</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btton_ok">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="chequeo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>591</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="termp_defecto_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Celsius:</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp_Celsius">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>671</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
